lib/src/server/handler.dart

```diff
@@ -61,9 +62,14 @@ class ServerHandlerImpl extends ServiceCall {
   Timer? _timeoutTimer;
   final X509Certificate? _clientCertificate;
 
-  ServerHandlerImpl(this._serviceLookup, this._stream, this._interceptors,
-      this._codecRegistry,
-      [this._clientCertificate]);
+  ServerHandlerImpl(
```

Comment:
Could you convert this one to use named parameter as well, so that call-sites are more readable?

Reply:
Done 😄